uiowa/uiowa
# Updated IISC projects view to sort by created date first

Resolves https://github.com/uiowa/uiowa/issues/6071. 
Resolves https://github.com/uiowa/uiowa/issues/6072. 

# How to test

- `ddev blt ds --site=iisc.uiowa.edu`
- Go to https://iisc.uiowa.ddev.site/projects
- Verify that default sort of the view is created date first, then alphabetical. 
- Verify that the Published date and academic Year display in the teaser view. 
- Apply filters and verify that sort remains the same. 
- Go to full project page node view and verify that published date is not appearing in layout. 

Updated screenshot:

<img width="1391" alt="Screenshot 2023-04-19 at 12 55 22 PM" src="https://user-images.githubusercontent.com/1036433/233159974-b27bcbd7-fe6f-49f0-ba5a-4c7e8201ac2f.png">
